exporter/coralogixexporter/README.md

```diff
@@ -48,40 +42,55 @@ exporters:
     # (Optional) Timeout is the timeout for every attempt to send data to the backend.
     timeout: 30s
 ```
-### Tracing deprecation 
 
-The v0.67 version removed old Jaeger based tracing endpoint in favour of Opentelemetry based one.
+### v0.76.0 Coralogix Domain 
 
-To migrate, please remove the old endpoint field, and change the configuration to `traces.endpoint` using the new Tracing endpoint.
+Since v0.76.0 you can specify Coralogix domain in the configuration file instead of specifying different endpoints for traces, metrics and logs. For example, the configuration below, can be replaced with domain field:
```

Comment:
Similarly to what was noted https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/20854/files#r1162958099, if it is a valid configuration to specify both `domain` and individual endpoints, it would be good to document which one takes a priority.